R/data.table.R

```diff
@@ -1,4 +1,12 @@
 
+if (!exists("trimws", "package:base")) {
```

Comment:
better as 

```
with(R.version, sprintf('%s.%s', major, minor)) >= '3.2.0'
```
?

Reply:
Why better?  You mean `if (getRversion() >= '3.2.0')` is the way to do that.  The comparison method for the `package_version` class does the right thing.

Reply:
good to know!